packages/broken/broken.0.4.2/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "1.2"
+version: "0.4.2"
+maintainer: "michipili@gmail.com"
+authors: "Michael Grünewald"
+license: "CeCILL-B"
```

Comment:
TL;DR: Some lawyers think that GPL and BSD are not valid in the French right, which actually exposes the author of the software to *unlimited* liability.

In the opinion of lawyers working for French CNRS, CEA and INRIA, licenses denying all liability are void in the French law, so that the author of a software published could actually (and hypothetically, since it seemingly never happened until this day) be sued in France because of the damage caused by the software. This would be very bad, because the damage would be liable just like any ordinary damage according to the Napoléon Code (just “code civil” in French :) CeCILL-B and its family transpose the three common licences model GPL, LGPL and CeCILL in the French right.

Since I have no competence in this domain I have no option but gathering informations and trust someone else's expertise. :)

That said, the situation is highly hypothetical since there is little – if any – court cases involving free software licences and especially their liability clause.

If you are curious about it, I can only suggest to spend your next coffee break reading the [FAQ](http://www.cecill.info/faq.en.html) it is as enjoyable and interesting as such a thing can be.  See also [that short overview](http://www.cecill.info/objectifs.en.html).